lib/improver/tests/utilities/test_DayNightMask.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
```

Comment:
This file should be in a subdirectory so that it is in `tests/utilities/solar/test_DayNightMask.py`. I'm pretty sure this is our convention (see https://github.com/metoppv/improver/wiki/Code-Style-Guide#unit-test-structure). Several other test files in utilities are also in the wrong place, but that doesn't mean we should continue with this.

Reply:
I isn't in the style guide but I will add it and a subdirectory solar